README.md

```diff
@@ -1,122 +1,146 @@
+
+## Purpose
+Flaky tests impact engineering velocity, reduce faith in test reliablity and give a false representation of code coverage. Manging flaky tests is a clunky process that involves constant build monitorization, difficult diagnosis and manual ticket creation. As a result, here at Flexport, we have created a Gem to automate the entire process to help improve the workflow and keep our massive test suites in prestine condition.
+
+The workflow at Flexport involves:
 
-bundle exec quarantine_dynamodb --aws_region us-west-1  # create the tables in us-west-1 in aws dynamodb
-                                                        # with default table names "quarantine_list" and
-                                                        # and "master_failed_tests"
+![ideal workflow](misc/flexport_workflow.png)
+
+---
+## Installation and Setup
+
+Add these lines to your application's Gemfile:
+```
+group :test do
+  gem 'quarantine'
+  gem 'rspec-retry
+end
 ```
 
-2. In your `spec_helper.rb` include the quarantine and rspec-retry gem
+And then execute:
 ```
-require "quarantine"
-require "rspec-retry"
+bundle install
 ```
-3. Configure rspec-retry to retry failed tests
+(might wanna move this to a later step)
+Create tables in your database required for the gem the upload and pull quarantined test info
+
+
+In your `spec_helper.rb` setup quarantine and rspec-retry gem. Click [rspec-retry](https://github.com/NoRedInk/rspec-retry) to get a more detailed explaination on rspec-retry configurations and how to setup.
 ```
-config.around(:each) do |example|
-  example.run_with_retry(retry: 2)
+require 'quarantine'
+require 'rspec-retry'
+
+Quarantine.bind({database: :dynamodb, aws_region: 'us-west-1'})
+
+RSpec.configure do |config|
+
+  config.around(:each) do |ex|
+    ex.run_with_retry(retry: 3)
+  end
+
 end
 ```
-4. Bind quarantine to RSpec in a CI environment
+
+Consider wrapping `Quarantine.bind` in if statements so local flaky tests don't pollute the list of quarantined tests
+
 ```
-if ENV[CI] && ENV[branch] == "master"
-  Quarantine.bind
+if ENV[CI] && ENV[BRANCH] == "master"
+  Quarantine.bind({database: :dynamodb, aws_region: 'us-west-1'})
 end
 ```
-5. Setup two tables in dynamodb in the correct region with the correct fields
-```
-@TODO: script to create tables in region
+
+Setup tables in AWS DynamoDB to support pulling and uploading quarantined tests
 ```
+bundle exec quarantine_dynamodb -h    # see all options
 
+bundle exec quarantine_dynamodb \     # create the tables in us-west-1 in aws dynamodb
+  --aws_region us-west-1              # with "quarantine_list" and "master_failed_tests"
+                                      # table names
+```
 
-## How to Set-up Development Locally
+You are all set to start quarantining tests!
 
-1. Setup dynamodb table names in AWS and add configurations
+## Try Quarantining Tests Locally
+Add a test that will flake consistently
```

Comment:
"flake consistently" is a bit of an oxymoron

Reply:
I agree they contradict. I removed `consistently`.